src/ngx_http_set_quote_sql.c

```diff
@@ -336,3 +336,75 @@ ngx_http_set_misc_escape_sql_str(u_char *dst, u_char *src, size_t size)
     return (uintptr_t) dst;
 }
 
+
+#ifndef NGX_LUA_NO_FFI_API
+int
+ngx_http_lua_ffi_set_misc_escape_sql_str(u_char *dst, const u_char *str,
```

Comment:
It's unacceptable to add `ngx_http_lua_ffi_xxx` functions into this `ngx_set_misc` module, which should have nothing to do with Lua by itself.